docs/cpp/configure-intellisense.md

```diff
@@ -4,63 +4,67 @@ Area: cpp
+A compiler is the only requirement to configure core IntelliSense functionality. To identify a compiler for IntelliSense, the C/C++ extension scans common paths on your machine for compilers such as Clang, GCC, MinGW, cygwin, cygwin64, and MSVC. If any of these compilers are identified and in a secure location, they are automatically configured for IntelliSense. Otherwise, a notification is displayed asking you to confirm that this compiler should be configured for IntelliSense. In either of these scenarios, the compiler selected is also set as the default compiler.
 
 ## How to check whether IntelliSense is configured
 
-If you don't have IntelliSense configured, the C/C++ Extension shows a yellow indicator in the status bar with a warning sign labeled **Configure IntelliSense**.
+If you don't have IntelliSense configured, the C/C++ extension shows a yellow indicator in the status bar with a warning sign labeled **Configure IntelliSense**.
 
-![configure-intelliSense-indicator-status-bar](images/intellisense/configure-intellisense-indicator.png)
+![Configure IntelliSense indicator in the status bar](images/intellisense/configure-intellisense-indicator.png)
 
-To configure, select the status bar indicator, which navigates you to the [configuration quick pick](/docs/cpp/configure-intellisense.md#_Option-1.-Select-a-configuration-option-through-the-configuration-quick-pick). The quick pick can help you select or install a C/C++ compiler.
+To configure, select the status bar indicator, which brings up the [configuration Quick Pick](#option-1-select-a-configuration-option-through-the-configuration-quick-pick). The Quick Pick can help you select or install a C/C++ compiler.
 
-If you do not see a status bar indicator, you can also check your project's `c_cpp_properties.json` file. This file stores all of your IntelliSense configuration settings. Navigate to this file by selecting **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`). Check the `IntelliSense mode` to find your configuration. To learn more about the `c_cpp_properties.json` file, see the [schema reference](/docs/cpp/c-cpp-properties-schema-reference.md).
+If you do not see a status bar indicator, you can also check your project's `c_cpp_properties.json` file. This file stores all of your IntelliSense configuration settings. Navigate to this file by selecting **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`). Check the `IntelliSense mode` to find your configuration. To learn more about the `c_cpp_properties.json` file, you can review the [schema reference](/docs/cpp/c-cpp-properties-schema-reference.md).
 
-![Command Palette](images/cpp/command-palette.png)
+![Command Palette filtered on C/C++ extension commands](images/cpp/command-palette.png)
```

Comment:
When looking at the staged version, we can actually make this image a lot shorter, could we use this instead? 
![image](https://github.com/microsoft/vscode-docs/assets/67650598/e429077e-ce1d-4620-b963-d75fe07761ee)